build/gen_stub.php

```diff
@@ -5808,7 +5853,31 @@ function replaceMethodSynopses(
 
             // Check if there is any change - short circuit if there is not any.
 
-            if (replaceAndCompareXmls($doc, $methodSynopsis, $newMethodSynopsis)) {
+            $isUnmodified = replaceAndCompareXmls($doc, $methodSynopsis, $newMethodSynopsis);
+
+            // Update deprecation warning
+
+            if ($funcInfo->deprecatedSince) {
+                foreach ($doc->getElementsByTagName("refentry") as $refentryElement) {
+                    foreach ($refentryElement->getElementsByTagName("refnamediv") as $refnameDivElement) {
+                        if (count($refnameDivElement->getElementsByTagName("refname")) !== 1) {
```

Comment:
The warning saying `This function has been DEPRECATED as of ...` doesn't make sense on pages where multiple functions/methods are documented, so I skip the automatic addition of the box in such cases.